src/mds/mdstypes.cc

```diff
@@ -285,12 +286,15 @@ void inode_t::encode(bufferlist &bl, uint64_t features) const
   ::encode(last_scrub_version, bl);
   ::encode(last_scrub_stamp, bl);
 
+  if (features & CEPH_FEATURE_FS_BTIME)
+    ::encode(btime, bl);
```

Comment:
Would an OSD ever even request anything here? This is an MDS call so I wouldn't think an OSD would ever see this reply. Also, if we encode unconditionally, won't that throw off the alignment of trailing info in the receiver? How do clients that don't have the btime patches know to skip over the btime in the stream?